lib/chef/resource/powershell_script.rb

```diff
@@ -46,6 +44,8 @@ def convert_boolean_return(arg=nil)
       def self.get_default_attributes(opts)
         {:convert_boolean_return => true}
       end
+
+      private_class_method :get_default_attributes
```

Comment:
it actually wasn't, it was public before.  the access modifiers don't apply to class methods defined like `def self.foo`, you have to `class << self` it or use the explicit declarations.  there's also no explicit `protected_class_method :foo` that just results in undefined method.